File: scratch/find_in_file.js
const fs = require('fs');
const path = require('path');

const filepath = "c:\\Users\\윤교마녀\\OneDrive - (주)예누\\vive-yenu\\blog\\src\\app\\page.js";
const query = process.argv[2];

if (!query) {
  console.log("Please provide a query.");
  process.exit(1);
}

console.log(`Searching for '${query}' in page.js...`);
const content = fs.readFileSync(filepath, 'utf-8');
const lines = content.split('\n');

lines.forEach((line, index) => {
  if (line.toLowerCase().includes(query.toLowerCase())) {
    console.log(`${index + 1}: ${line.trim()}`);
  }
});

File: scratch/find_in_file.py
import sys
import re

filepath = r"c:\Users\윤교마녀\OneDrive - (주)예누\vive-yenu\blog\src\app\page.js"

query = sys.argv[1] if len(sys.argv) > 1 else ""
if not query:
    print("Please provide a query.")
    sys.exit(1)

print(f"Searching for '{query}' in {filepath}...")
with open(filepath, "r", encoding="utf-8") as f:
    for idx, line in enumerate(f, 1):
        if re.search(query, line, re.IGNORECASE):
            print(f"{idx}: {line.strip()}")
